src/assets/i18n/fr.json

```diff
@@ -117,55 +117,55 @@
-  "Delete zvol device": "",
-  "Delete {name}?": "",
+  "Delete zvol device": "Supprimer un périphérique zvol",
+  "Delete {name}?": "Supprimer {name} ?",
   "Description": "",
   "Destination": "",
   "Details": "",
   "Determine how  <a href=\"https://man7.org/linux/man-pages/man1/chmod.1.html\" target=\"_blank\">chmod</a>  behaves when adjusting file ACLs. See the  <a href=\"https://linux.die.net/man/8/zfs\" target=\"_blank\">zfs(8)</a> aclmode property.  <br><br><i>Passthrough</i> only updates ACL entries that are related to the file or directory mode.  <br><br><i>Restricted</i> does not allow chmod to make changes to files or directories with a  non-trivial ACL. An ACL is trivial if it can be fully expressed as a file mode without losing  any access rules. Setting the ACL Mode to Restricted is typically used to optimize a dataset for  SMB sharing, but can require further optimizations. For example, configuring an rsync task with this  dataset could require adding <i>--no-perms</i> in the task <i>Auxiliary Parameters</i> field.": "",
   "Determines the outgoing and incoming traffic ports.<br>  <i>LACP</i> is the recommended protocol if the network switch is capable of  active LACP.<br><i>Failover</i> is the default protocol choice and  should only be used if the network switch does not support active LACP.": "",
-  "Device Busy": "",
-  "Device names of each disk being edited.": "",
+  "Device Busy": "Dispositif occupé",
+  "Device names of each disk being edited.": "Noms des périphériques de chaque disque en cours d'édition.",
   "Direction": "",
-  "Directory Inherit": "",
-  "Disable AD User / Group Cache": "",
-  "Discard": "",
-  "Disk Sector Size": "",
-  "Disk device name.": "",
-  "Disk not attached to any pools.": "",
-  "Disks Overview": "",
-  "Dismissed": "",
-  "Docker Images": "",
+  "Directory Inherit": "Héritage du répertoire",
+  "Disable AD User / Group Cache": "Désactiver le cache Utilisateur/Groupe AD",
+  "Discard": "Rejeter",
+  "Disk Sector Size": "Taille secteur disque",
+  "Disk device name.": "Nom du périphérique de disque.",
+  "Disk not attached to any pools.": "Le disque n'est attaché à aucun volume.",
+  "Disks Overview": "Présentation des disques",
+  "Dismissed": "Rejeté",
+  "Docker Images": "Images Docker",
   "Does your business need <a href=\"https://www.ixsystems.com/truenas/?utm_source=truenas+core+ui&utm_medium=os&utm_campaign=welcome\" target=\"_blank\">Enterprise level</a> support and services? Contact <a href=\"https://www.ixsystems.com/quote-form/?utm_source=truenas+core+ui&utm_medium=os&utm_campaign=welcome\"target=\"_blank\">iXsystems</a> for more information.": "",
-  "Download Updates": "",
-  "Download actions": "",
-  "Drive Details": "",
-  "Drive Temperatures": "",
+  "Download Updates": "Télécharger les mises à jour",
+  "Download actions": "Actions de téléchargement",
+  "Drive Details": "Détails du lecteur",
+  "Drive Temperatures": "Températures du lecteur",
```

Comment:
Google translate shows this doesn't exactly translate to what the text says. Please reconsider. Same goes for other similar strings.

Reply:
OK, done

Reply:
Hello, is it possible to validate in order to continue the translation?